pkg/osquery/tables/crowdstrike/falcon_kernel_check/table.go

```diff
@@ -0,0 +1,89 @@
+package falcon_kernel_check
+
+import (
+	"context"
+	"regexp"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/kolide/launcher/pkg/osquery/tables/tablehelpers"
+	"github.com/osquery/osquery-go/plugin/table"
+	"github.com/pkg/errors"
+)
+
+const kernelCheckUtilPath = "/opt/CrowdStrike/falcon-kernel-check"
+
+type Table struct {
+	logger log.Logger
+}
+
+func TablePlugin(logger log.Logger) *table.Plugin {
```

Comment:
Not a blocker, but since this is probably going to share a directory with another table, we should name these slightly more descriptively.

I'm also not sure we need to export `Table`